backend/src/receipts.js

```diff
@@ -61,6 +61,18 @@ async function getReceiptsList(blockHash) {
   return receipts;
 }
 
+// As a temporary solution we split receipts list into two lists:
```

Comment:
What would be a permanent solution?

Reply:
One of possible solutions is here https://github.com/near/near-explorer/pull/868. I'll add a link to this PR to this comment